addon/styles/_pix-select.scss

```diff
@@ -1,70 +1,159 @@
 .pix-select {
-  @import 'reset-css';
-
+  display: inline-block;
   position: relative;
 
-  @mixin pixSelect {
-    appearance: none;
-    border: 1px $pix-neutral-40 solid;
-    border-radius: 4px;
-    color: $pix-neutral-90;
-    font-family: $font-roboto;
-    font-size: 0.875rem;
-    height: 36px;
-    width: 100%;
-    text-overflow: ellipsis;
-
-    @include hoverFormElement();
-    @include focusFormElement();
-    @include focusWithinFormElement();
+  &__label {
+    @include label();
   }
 
-  select {
-    @include pixSelect;
-    padding: 0 32px 0 16px;
+  &__sub-label {
+    @include subLabel();
+  }
 
-    &.pix-select--big {
-      height: 44px;
+  &__dropdown {
+    z-index: 200;
+    background-color: $pix-neutral-0;
+    position: absolute;
+    border-top: none;
+    border-radius: 0 0 4px 4px;
+    overflow-y: auto;
+    max-height: 200px;
+    list-style-type: none;
+    padding: 0;
+    box-shadow: 0px 8px 24px 0px rgba(23, 43, 77, 0.1);
+    transition: all .1s ease-in-out;
+    white-space: nowrap;
```

Comment:
z-index: 200;
    position: absolute;
    border-top: none;
    border-radius: 0 0 4px 4px;
    overflow-y: auto;
    max-height: 200px;
    list-style-type: none;
    padding: 0;
    background-color: $pix-neutral-0;
    box-shadow: 0px 8px 24px 0px rgba(23, 43, 77, 0.1);
    transition: all .1s ease-in-out;
    white-space: nowrap;